src/modules/roc_audio/resampler_updater.h

```diff
@@ -35,7 +35,8 @@ class ResamplerUpdater : public packet::IWriter,
     //!  - @p update_interval defines how often to call FreqEstimator, in samples
     //!  - @p aim_queue_size defines FreqEstimator target queue size, in samples
     ResamplerUpdater(packet::timestamp_t update_interval,
-                     packet::timestamp_t aim_queue_size);
+                     packet::timestamp_t aim_queue_size,
+                     const float sample_rate_coef);
```

Comment:
sample_rate_coef parameter should be documented.

Reply:
Fxd